src/state/stateSingle.test.ts

```diff
@@ -41,7 +41,7 @@ describe("stateSingle", () => {
       scheduler().run(({ expectObservable, expectSubscriptions, cold }) => {
```

Comment:
I was reading through these tests and added a few tests and assertions to aid my own understanding of `state()`.